static/app/bootstrap/initializeSdk.tsx

```diff
@@ -58,6 +59,18 @@ function getSentryIntegrations(sentryConfig: Config['sentryConfig'], routes?: Fu
     }),
   ];
 
+  if (sentryConfig.replaysSessionSampleRate || sentryConfig.replaysOnErrorSampleRate) {
+    integrations.push(
```

Comment:
Now that we push this here, do we need to remove it from elsewhere? Since Replay is currently working, I believe it was added in some other way, right?

Reply:
@Lms24's
> Looks good to me on first glance. I guess we'd need to adapt getsentry to delete the custom Replay intialization but overall, this would allow us to dogfood the Replay integration more closely to how users mostly use Replay.

do we need those two changes in at the same time to land on the same release?

Reply:
Really not an expert on the Sentry app, but my tentative answer would be "ideally yes", as else we'd probably initialize Replay twice for a certain period of time 🤔

Reply:
Probably the way to go here is:

1. Update getsentry in preparation for this, and merge that change in. That means removing any replay-specific SDK setup there, and just making sure the sample rate config is set. Merge this in - temporarily, replay will be disabled, until...
2. We merge this change, which will make replay work again on getsentry.

Does that make sense? I think that's the easiest way to go.

Reply:
Sounds good to me!

Reply:
@billyvg shared a relevant detail:

> Note that we are sampling via flagr and this would not work with what we are doing. We have more granular sampling with flagr. Eg different sample rates between employees and normal users

Reply:
@mydea imo we should keep sampling with flagr, it gives us much better control over sampling and without having to make code changes.

Reply:
Another reason for us to look at dynamic sampling for Replay as customers will have the same thought/need

Reply:
Yeah, though another reason we used flagr was to avoid loading the replay sdk completely if user was not sampled. (This was also important early on to be able to quickly turn loading replay SDK off in when we initially added replay to Sentry SaaS, though less relevant now since the SDK has been tested and is stable).

Reply:
I wonder if there is a way to build this into replay, as I guess if that's important for us, it might be important for other users as well. 

But for now, I've implemented this differently, so we can close this for now!